packages/cozy-harvest-lib/src/datacards/FileDataCard.jsx

```diff
@@ -0,0 +1,120 @@
+import ListItem from '@material-ui/core/ListItem'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import ListItemText from '@material-ui/core/ListItemText'
+
+import palette from 'cozy-ui/transpiled/react/palette'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
+import Circle from 'cozy-ui/transpiled/react/Circle'
+import FileIcon from 'cozy-ui/transpiled/react/Icons/File'
+
+import CozyClient, {
+  Q,
+  queryConnect,
+  isQueryLoading,
+  hasQueryBeenLoaded
+} from 'cozy-client'
+
+import Card from 'cozy-ui/transpiled/react/Card'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { getFileIcon } from './mime-utils'
+
+const LoadingFileListItem = ({ divider }) => {
+  return (
+    <ListItem divider={divider}>
+      <ListItemIcon>
+        <Skeleton variant="circle" />
+      </ListItemIcon>
+      <ListItemText primary={<Skeleton variant="avatar" />} />
+    </ListItem>
+  )
+}
+
+const FileCard = ({ files, loading, konnector }) => {
+  const { t } = useI18n()
+  return (
+    <Card className="u-ph-0 u-pb-0 u-ov-hidden">
+      <div className="u-ph-1 u-mb-half">
+        <Media align="top">
+          <Img>
+            <Circle
+              size="small"
+              backgroundColor={palette.puertoRico}
+              className="u-mr-1"
+            >
+              <Icon icon={FileIcon} color={palette.white} />
+            </Circle>
+          </Img>
+          <Bd>
+            <Typography variant="h5">{t('datacards.files.title')}</Typography>
+            <Typography variant="caption">
+              {t('datacards.files.caption', { konnectorName: konnector.name })}
+            </Typography>
+          </Bd>
+        </Media>
+      </div>
+      <List>
+        {loading ? (
+          <>
+            <LoadingFileListItem divider />
+            <LoadingFileListItem divider />
+            <LoadingFileListItem divider />
+            <LoadingFileListItem divider />
+            <LoadingFileListItem />
+          </>
+        ) : (
+          files.map((file, i) => (
+            <ListItem key={file._id} divider={i !== files.length - 1}>
+              <ListItemIcon>
+                <Icon icon={getFileIcon(file)} width="32" height="32" />
+              </ListItemIcon>
+              <ListItemText primary={file.name} />
+            </ListItem>
+          ))
+        )}
+      </List>
+    </Card>
+  )
+}
+
+const makeQueryFromProps = ({ accountId }) => ({
+  query: Q('io.cozy.files')
+    .where({
+      'cozyMetadata.sourceAccount': accountId
+    })
+    .sortBy([
+      { 'cozyMetadata.sourceAccount': 'desc' },
+      { 'cozyMetadata.createdAt': 'desc' }
+    ])
+    .indexFields(['cozyMetadata.sourceAccount', 'cozyMetadata.createdAt'])
+    .limitBy(5),
+  as: `io.cozy.accounts/${accountId}/io.cozy.files`,
+  fetchPolicy: CozyClient.fetchPolicies.olderThan(30 & 1000)
```

Comment:
Is it really wanted to use this operator? Seems like a typo

Reply:
Done